src/schema/artwork/index.ts

```diff
@@ -535,13 +535,14 @@ export const artworkFields = () => {
         ).resolve(artwork, { precision: 0 })
 
         if (domesticShipping && !internationalShipping)
-          return "Shipping: " + domesticShipping + " Continental US only"
+          return "Shipping: " + domesticShipping + " continental US only"
+        if (!domesticShipping) domesticShipping = "Free"
         return (
           "Shipping: " +
           domesticShipping +
-          " Continental US, " +
+          " continental US, " +
           internationalShipping +
-          " rest of the world"
+          " rest of world"
```

Comment:
Designs show `rest of world` in the text instead of `rest of the world`... Even though the later one sounds more proper to me I'm by no means an expert on English, so updating to designs.

Reply:
Yea, `rest of the world` sounds better if one were speaking, but I guess a shorter text-only version snippet `rest of world` also works? But I agree with you!